x/storage/types/message.go

```diff
@@ -1661,6 +1661,11 @@ func (msg *MsgUpdateObjectContent) ValidateBasic() error {
 		return err
 	}
 
+	err = s3util.CheckValidContentType(msg.ContentType)
```

Comment:
Isn't this a hardfork logic?

Reply:
this `MsgUpdateObjectContent` is not in testnet yet